### PR TITLE
Added queues for logistics tasks

### PR DESCRIPTION
**Pull request by kkrampa**
@czue

### CODE REVIEW

File: fab/fabfile.py [hunk 1]
```diff
@@ -1072,6 +1072,8 @@ def set_celery_supervisorconf():
         'saved_exports_queue':          ['supervisor_celery_saved_exports_queue.conf'],
         'ucr_queue':                    ['supervisor_celery_ucr_queue.conf'],
         'email_queue':                  ['supervisor_celery_email_queue.conf'],
+        'logistics_reminder_queue':     ['supervisor_celery_logistics_reminder_queue.conf'],
```
Comment (dimagimon) [inline fab/fabfile.py]: E241 multiple spaces after ':'

File: fab/fabfile.py [hunk 2]
```diff
@@ -1072,6 +1072,8 @@ def set_celery_supervisorconf():
...
+        'logistics_background_queue':   ['supervisor_celery_logistics_background_queue.conf'],
```
Comment (dimagimon) [inline fab/fabfile.py]: E241 multiple spaces after ':'

**Comment by czue** [comment]
@gcapalbo this adds 4 total queue jobs to celery1. do you think that's ok? here's the last 7 days of memory usage - so usually around 60% but some occassional red spikes.

image

**Comment by gcapalbo** [comment]
@czue yup, should be fine

**Pull request merged**